runtime/near-vm-logic/src/tests/miscs.rs

```diff
@@ -237,22 +226,21 @@ fn test_log_total_length_limit_mixed() {
 #[test]
 fn test_log_utf8_max_limit_null_terminated() {
     let mut logic_builder = VMLogicBuilder::default();
-    let mut string_bytes = "j ñ r'ø qò$`5 y'5 øò{%÷ `Võ%".as_bytes().to_vec();
-    let limit = (string_bytes.len() - 1) as u64;
+    let bytes = "j ñ r'ø qò$`5 y'5 øò{%÷ `Võ%".as_bytes();
+    let limit = (bytes.len() - 1) as u64;
     logic_builder.config.limit_config.max_total_log_length = limit;
     let mut logic = logic_builder.build(get_context(vec![], false));
+    let bytes = logic.internal_mem_write(bytes);
```

Comment:
Same as above, except here in addition you’re relying on the fact that the mock memory starts all-0, which sounds even worse to me 😅

Reply:
> you’re relying on the fact that the mock memory starts all-0, which sounds even worse to me

That is a contract of MemoryLike.  They always start initialised with zeros.